lib/omniauth/strategies/mediawiki.rb

```diff
@@ -43,8 +47,13 @@ def request_phase
         end
 
         r.finish
+      rescue RequestTokenError => e
+        puts "There was a problem in getting the token from Mediawiki, here's what Mediawiki returned:"
+        puts request_token.inspect
```

Comment:
This is going to be a fatal NoMethodError somewhere else if this returns nil. We may as well re-raise the error after printing the debug info.